src/pages/home/ReportScreen.js

```diff
@@ -225,19 +225,19 @@ class ReportScreen extends React.Component {
         const animatePlaceholder = !freeze;
 
         return (
-            <Freeze
-                freeze={freeze}
-                placeholder={(
-                    <ScreenWrapper
-                        style={screenWrapperStyle}
-                    >
-                        <ReportHeaderSkeletonView animate={animatePlaceholder} />
-                        <ReportActionsSkeletonView animate={animatePlaceholder} containerHeight={this.state.skeletonViewContainerHeight} />
-                    </ScreenWrapper>
-                )}
+            <ScreenWrapper
+                style={screenWrapperStyle}
             >
-                <ScreenWrapper
-                    style={screenWrapperStyle}
+                <Freeze
+                    freeze={freeze}
+                    placeholder={(
+                        <ScreenWrapper
+                            style={screenWrapperStyle}
+                        >
+                            <ReportHeaderSkeletonView animate={animatePlaceholder} />
+                            <ReportActionsSkeletonView animate={animatePlaceholder} containerHeight={this.state.skeletonViewContainerHeight} />
+                        </ScreenWrapper>
```

Comment:
We can remove the `ScreenWrapper` entirely now from the placeholder and just use a [fragment](https://reactjs.org/docs/fragments.html).

Reply:
I think this will fix the padding issue too? 🙏